addons/website/models/ir_ui_menu.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, tools
+from odoo.http import request
+
+
+class IrUiMenu(models.Model):
+    _inherit = 'ir.ui.menu'
+
+    @api.model
+    @tools.ormcache_context('self._uid', keys=('lang', 'force_action',))
+    def load_menus_root(self):
+        root_menus = super().load_menus_root()
+        if self._context.get('force_action'):
+            web_menus = self.load_web_menus(request.session.debug if request else False)
+            for menu in root_menus['children']:
+                # Force the action.
+                if (
+                    self._context.get('force_action')
```

Comment:
You already checked that in the above `if`

Reply:
Indeed, removed